tests/adapter/dbt/tests/adapter/constraints/fixtures.py

```diff
@@ -196,6 +239,95 @@
           - type: primary_key
           - type: check
             expression: (id > 0)
+          - type: check
+            expression: id >= 1
+        tests:
+          - unique
```

Comment:
question to myself - why is this a test and not a constraint?

Reply:
answer: copying it over from[ the existing fixture](https://github.com/dbt-labs/dbt-core/blob/main/tests/adapter/dbt/tests/adapter/constraints/fixtures.py#L238)

A test for a unique constraint has also been added in this PR: https://github.com/dbt-labs/dbt-core/pull/7512/files#diff-acc8cb2ee65ad9cfdd7f4555ac47b95eb4443413ac3a81c8dc913aae5eb7bc34R330